portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,6 +19,7 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
```

Comment:
This comment should reflect what the servlet actually does (e.g. something about how it returns comments as a JSON list).

Reply:
oh ok. I though I left a comment but I will now